x-pack/platform/plugins/shared/fleet/public/hooks/use_request/epm.ts

```diff
@@ -110,6 +110,10 @@ export const useGetPackagesQuery = (
     enabled: options?.enabled,
     retry: (_, error) => !isRegistryConnectionError(error),
     refetchOnWindowFocus: false,
+    select: (data) => {
+      // do not display the security_ai_prompts package in the UI
+      return { items: data.items?.filter((item) => item.id !== 'security_ai_prompts') };
```

Comment:
we have some filtering logic for the cards here:
https://github.com/elastic/kibana/blob/c4160b1c4048f236e3b29226ab2dd20b66d62d42/x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/home/hooks/use_available_packages.tsx#L177
(this is currently used to hide agentless-only packages when agentless is not available on the deployment)

I would love to see the filtering for `security_ai_prompts` added there, instead of another place where filtering logic exists. wdyt?

Reply:
@jen-huang ah, I've just realized I can still search for it in the search bar and get to the page. Do you know if there is a way i can erase any evidence of it from the kibana UI?

Reply:
@stephmilovic What if we filter it out on the server-side, in the handler for list packages API (`/api/fleet/epm/packages`)?:
https://github.com/elastic/kibana/blob/62a4cc620ea157505d4715a7cb0ad56a56e10512/x-pack/platform/plugins/shared/fleet/server/routes/epm/handlers.ts#L108

That should handle the integrations list (installed & uninstalled) UI, as well as filtering it out from the Kibana global search, since it's using the same endpoint.

This assumes that Security Assistant isn't relying on this API for this package :)